google-cloud-pubsublite/src/main/java/com/google/cloud/pubsublite/internal/wire/PublisherBuilder.java

```diff
@@ -42,9 +41,9 @@
  * Publisher<Offset> publisher = PublisherBuilder.builder()
  *   .setTopic(
  *     TopicPaths.newBuilder()
- *       .setProjectNumber(MY_PROJECT)
- *       .setTopicName(MY_TOPIC)
- *       .setZone(CloudZone.of("us-east1-a"))
+ *       .setProject(MY_PROJECT)
```

Comment:
Should this be

.setProject(ProjectName.of(MY_PROJECT))

?

Reply:
Nope. ProjectName isnt a thing. ProjectId is a thing, but the method is overloaded to accept a ProjectNumber, ProjectId, or ProjectIdOrNumber